Tests/ArcGISToolkitTests/PreplannedMapModelTests.swift

```diff
@@ -169,10 +169,14 @@ class PreplannedMapModelTests: XCTestCase {
         let areas = try await task.preplannedMapAreas
         let area = try XCTUnwrap(areas.first)
         let areaID = try XCTUnwrap(area.portalItem.id)
-        let mmpkDirectory = FileManager.default.preplannedDirectory(
```

Comment:
I refactored to make the `FileManager.preplannedDirectory` a private method:

- If the component is self contained, then a developer shouldn't touch the file structure that the component maintains
- It should be like a black box that all the offline maps are managed internally, and the developer can only access the maps via the APIs we provide them